terraform-post-packer/variables.tf

```diff
@@ -9,3 +9,9 @@ variable "extraorg_account_ids" {
   description = "A list of AWS account IDs corresponding to \"extra\" accounts with which you want to share this AMI (e.g. [\"123456789012\"]).  Normally this variable is used to share an AMI with accounts that are not a member of the same AWS Organization as the account that owns the AMI."
   default     = []
 }
+
+variable "recent_ami_count" {
+  type        = number
+  description = "The number of most-recent AMIs for which to grant launch permission (e.g. \"3\").  If this variable is set to three, for example, then accounts will be granted permission to launch the three most recent AMIs."
```

Comment:
Do we want to put any language in here about the case where there are fewer existing AMIs than `recent_ami_count`?

Reply:
Please accept commit 77378c7 as tribute.